docs/monitors/gitlab-gitaly.md

```diff
@@ -36,6 +36,9 @@ Configuration](../monitor-config.md#common-configuration).**
 | `password` | no | `string` | Basic Auth password to use on each request, if any. |
 | `useHTTPS` | no | `bool` | If true, the agent will connect to the exporter using HTTPS instead of plain HTTP. (**default:** `false`) |
 | `skipVerify` | no | `bool` | If useHTTPS is true and this option is also true, the exporter's TLS cert will not be verified. (**default:** `false`) |
+| `caCertPath` | no | `string` | Path to the CA cert that has signed the TLS cert, unnecessary if `skipVerify` is set to false. |
+| `clientCertPath` | no | `string` | Path to the client TLS cert to use for TLS required connections |
```

Comment:
Ok, so for these monitors if the user sets these options and is also using HTTPS, then these configured client certs will be used automatically?

Reply:
Yes, if the certs are provided, it will be used by the http client in prometheus exporter